src/commcare_cloud/fab/utils.py

```diff
@@ -69,6 +70,14 @@ def repo(self):
         self._repo = github.get_organization('dimagi').get_repo('commcare-hq')
         return self._repo
 
+    @property
+    def last_commit_sha(self):
+        if self.last_tag:
+            return self.last_tag.commit.sha
+
+        with cd(env.code_current):
+            return sudo('git rev-parse HEAD')
```

Comment:
Oh interesting - I didn't even think of having the actual `current` dir in scope for this kind of thing - I assumed this all ran locally before you started talking to machines.  Does this happen on one particular target machine or something?

Reply:
Yeah. this only works because I used `execute` https://github.com/dimagi/commcare-cloud/pull/1755/files#diff-4b38b47cd9db17e5e8e82f5424318f83R170 which calls it on a remote server instead of your local machine. I believe it's happening on the one tagged 'deploy' which should also be 'web0'